view/frontend/templates/js/replacejs.phtml

```diff
@@ -377,55 +387,51 @@ window.boltConfig = <?=$this->getSettings(); ?>;
         // the Bolt order is created right after the checkout button is clicked
         // and the checkout modal popup is opened only if order creation was successfull
         check: function () {
-
             // if orderToken is set the checkout window will open
             return !!cart.orderToken;
         }
     };
     /////////////////////////////////
 
-    var create_request;
 
+    /////////////////////////////////////////////////////
+    // Create Bolt order and configure BoltConnect
+    /////////////////////////////////////////////////////
+    var create_request;
     var createOrder = function () {
-
         if (create_request) create_request.abort();
-
         // preset success conditioin to false
         cart.orderToken = '';
-
         // define the params sent with the request variable
         var params = [];
-
         // check and set payment_pnly flag
         if (getCheckoutType() === 'payment' && settings && !!settings.publishable_key_payment) {
             params.push('payment_only=true')
         }
-
         // get additional page data to be sent with the request,
         // one page checkout billing info, email, ...
         var place_order_payload = document.getElementById(place_order_payload_id);
         if (place_order_payload && place_order_payload.value) {
             params.push('place_order_payload=' + place_order_payload.value);
         }
-
         params = params.join('&');
-
         if (params === '') params = null;
-
         // set cart and hints data in a response callback
         var onSuccess = function(data){
             cart  = data.cart;
-            hints = data.hints;
+
+            var prefill = isObject(data.hints.prefill) ? deepMeggeObjects(hints.prefill, data.hints.prefill) : hints.prefill;
+            hints = deepMeggeObjects(hints, data.hints);
+            hints.prefill = prefill;
+
             hints.publishableKey = getCheckoutKey();
-            BoltConnect.process(cart, hints, callbacks);
-        };
 
-        var onError = function () {
+            BoltConnect.process(cart, hints, callbacks);
```

Comment:
Please use `BoltCheckout.configure` instead of `BoltConnect.process`. These two are basically same but we recently renamed.